### ISSUE TITLE
UI should display tale starting spinner during restart

### ISSUE THREAD

**Issue by craig-willis** [whole-tale/ngx-dashboard#215]
**Problem**
When restarting a tale, the interactive session/IDE can no longer connect to the remote container, displaying various errors. Ideally we would hide the IDE and display some indicator during restart like we do during start.

**To repeat**
* Create and start a Jupyter tale
* Create `apt.txt` with contents `vim`
* Rebuild tale
* While on interactive tab restart tale

**Expected results**
UI displays tale starting spinner

**Actual results**
UI displays IDE which can no longer connect to remote container resulting in various errors/warnings (e.g., Directory "" not found)

**Comment by bodom0015** [comment]
Is there an `InstanceStatus` change when restarting? If the current restart logic already sets the state back to Launching, I would think that the existing logic should handle this case already.

If not, aside from possibly watching for updates coming through the SyncService I'm not sure how to detect if the restart has completed? We could consider adding logic that uses the Launching + Running states to indicate restart, or add a new explicit state for `InstanceStatus.RESTARTING`

**Comment by craig-willis** [comment]
Good catch. It looks like the `update_container` task doesn't change the instance status. I'll look into fixing this in `girder_wholetale` or `gwvolman`.
